tests/FakeItEasy.IntegrationTests/FakingClassesTests.cs

```diff
@@ -12,10 +11,7 @@ public class FakingClassesTests
         [Test]
         public void Should_be_able_to_get_a_fake_value_of_uri_type()
         {
-            using (Fake.CreateScope(new NullFakeObjectContainer()))
-            {
-                A.Fake<Uri>();
-            }
+            A.Fake<Uri>();

```

Comment:
It should really be done in this kind of way:

```C#
var exception = Record.Exception(() => A.Fake<Uri>());

exception.Should().BeNull("because we can create fakes of type 'System.Uri'");
```